content/stm32/flash-cube-programmer.md

```diff
@@ -1,25 +1,47 @@
 # How To Flash a target using STM32 Cube Programmer
 
-This is a guide on how to manually flash the firmware using STM32CubeProgrammer utility.  This utility replaces the ST-Link utility and may be required for some Windows 10 machines.
+This is a "rough" guide on how to manually flash the firmware using STM32CubeProgrammer utility.
 
-You will need to use this utility to replace the boot loader on the STM32 chip.  Once replaced, you can do all further firmware maintentance using the nanoFramework Firmware Flash tool (nanoff.exe).
+NOTE: This utility replaces the deprecated ST-Link utility and is required as a prerquisite for boards with outdated firmware. Outdated boards will need to use this utility to update their current boot loader firmware. Once updated, all further firmware maintentance can be carried out using the nanoFramework Firmware Flash tool [nanoff](https://github.com/nanoframework/nanoFirmwareFlasher).
```

Comment:
```suggestion
NOTE 1: This utility replaces the deprecated ST-Link utility. 
NOTE 2: The recommended tool to install/update nanoFramework it's using the nanoFramework Firmware Flasher tool [nanoff](https://github.com/nanoframework/nanoFirmwareFlasher).
```

Reply:
I was splitting the comment on 2 notes.

Note 1 as an informative bit about the tool it's replacing (although that's not much relevant, especially for new comers)
Note 2 was to highlight that the preferred tool it's nanoff, not STCube programmer. This is is here mostly for historical reasons when nanoff didn't exist. With the current state, nanoff it's the goto tool. 

The original comments on the need to use this for boars with "outdated firmware" also have no meaning now as this is not the case anymore. That's why I remove them.